telethon/tl/tlobject.py

```diff
@@ -125,17 +129,21 @@ def serialize_datetime(dt):
             return b'\0\0\0\0'
 
         if isinstance(dt, datetime):
-            dt = int(dt.timestamp())
+            #If no tzinfo is provided then we assume dt is in local timezone
+            epoch_selected = _EPOCH_NATIVE_LOCAL if dt.tzinfo is None else _EPOCH
+            #We use .total_seconds() method instead of simply dt.timestamp(), 
```

Comment:
Can you add a space after the `#`? Half personal taste, half the fact it's the style used everywhere else in the library (and remove the trailing space):
```suggestion
            # We use .total_seconds() method instead of simply dt.timestamp(),
```